hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -18,6 +18,10 @@ private Hawk() {
     // no instance
   }
 
+  public enum Status {
+    NOT_INITIALISED, INITIALISED
```

Comment:
If we need status, then please use support annotations like IntDef or StringDef http://tools.android.com/tech-docs/support-annotations
Please don't use "enum".